packages/fuzzer/fuzzing_async.cpp

```diff
@@ -107,6 +117,10 @@ void CallJsFuzzCallback(Napi::Env env, Napi::Function jsFuzzCallback,
   // thread and continue with the next invocation.
 
   try {
+    if (setjmp(errorBuffer) != 0) {
```

Comment:
Could you please add a comment describing why this is handled in the addon's main thread?